pkgs/servers/home-assistant/default.nix

```diff
@@ -225,6 +225,7 @@ in with py.pkgs; buildPythonApplication rec {
     "conversation"
     "counter"
     "cover"
+    "debugpy"
```

Comment:
This can be conditionally added when `stdenv.hostPlatform.system` is supported by `debugpy`:
```nix
{
  componentTests = [
    ...
  ] ++ lib.optionals (builtins.any (s: s == stdenv.hostPlatform.system) debugpy.meta.platforms) [
    "debugpy"
  ];
}
```